### DISCUSSION TITLE
Whats the secret of the non buffered nearly realtime debug stream?

### DISCUSSION THREAD

**Discussion by corgan2222** [blakeblackshear/frigate#807] [Ask A Question]
For my automatic door opener on face detection, I tried any kind of software and technic to get the fastest face detection with a rtsp stream IP Camera. 
I tryed python, opencv, node, ffmpeg, aso. 

But all solutions have a delay from 20-60seconds which is annoying if someone has to wait in front of the door and wait for the buffer... 

The only real-time camera stream is the debug stream from frigate.
I looked into the code, but I'm not sure what's the "magic" deal here to avoid any kind of stream buffer.
thanks

**Comment by blakeblackshear** [comment]
Take a look at the ffmpeg command used by frigate. It's available at `/api/config`.
